.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "lib/sdai"]
 	path = lib/sdai
 	url = https://github.com/makerdao/sdai
+[submodule "lib/dss-lite-psm"]
```

Comment:
I actually wouldn't import this dependency in this case. I would just use an inline interface since none exists in this repo. You shouldn't be using the implementation for an interface on a fork.

Reply:
Removed